tests/run-make/llvm-location-discriminator-limit-dummy-span/rmake.rs

```diff
@@ -0,0 +1,58 @@
+//! Regression test for <https://github.com/rust-lang/rust/issues/135332>.
+//!
+//! We can't simply drop debuginfo location spans when LLVM's location discriminator value limit is
+//! reached. Otherwise, with `-Z verify-llvm-ir` and fat LTO, LLVM will report a broken module for
+//!
+//! ```text
+//! inlinable function call in a function with debug info must have a !dbg location
+//! ```
+
+//@ ignore-cross-compile
+//@ needs-dynamic-linking
+//@ only-nightly: requires unstable rustc flag
+
+#![deny(warnings)]
+
+use run_make_support::{dynamic_lib_name, rfs, rust_lib_name, rustc};
+
+// Generate a program that has a *lot*
```

Comment:
Suggestion:

```suggestion
// Synthesize program that requires a lot of location discriminators.
```

or some wording to that effect, or just remove this.